test/functional/apps/dashboard/new_visualize_flow/dashboard_embedding.ts

```diff
@@ -74,6 +80,9 @@ export default function ({ getService, getPageObjects }) {
         name: 'Embedding Markdown Test',
         markdown: 'Nice to meet you, markdown is my name',
       });
+      await PageObjects.visualize.saveVisualization('Embedding Markdown', {
+        redirectToOrigin: true,
+      });
```

Comment:
I had to add saving visualization (with `add to dashboard` checked option) in order to make both tests pass. Does it follow the expected behaviour?

Reply:
The purpose of this test is to verify the new "by-value" flow, which doesn't require visualizations to be saved. In order for it to work, the setting in dashboard config file needs to be set to true:
https://github.com/elastic/kibana/blob/master/src/plugins/dashboard/config.ts#L23

When I originally wrote this test, the feature flag was in the visualize plugin and it was called `showNewVisualizeFlow`. When I was moving the files to the new feature flag, I deleted that setting altogether, cause the tests were not running anyway:
https://github.com/elastic/kibana/commit/be0724dd91f4158bf58c1b019758beec0523c3e5#diff-01c99ab9d81cbd6f07e2612c757a4cdd34d3dedbf0434b23f6939fb7705677f9

I think the original test should be left as-is, with no modifications (as the behavior is still the same), but we'd need a config file with `dashboard.allowByValueEmbeddables=true`.